### PR TITLE
Favicon is added #193

### PR DESCRIPTION
**Pull request by Syed-Ansar**
Favicon is added #193.

Review Changes. @TildaDares.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,22 +50,22 @@ <h3>Quick start</h3>
...
                          <a class="btn btn-success" id="preset_ndvi_red_color">Colorized</a></td>
                    </tr>
                  </table>
-                 <p>Post a research note at <a href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
+                 <p>Post a research note at <a target="_blank"  href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
 
 	    </div>
             <div class="modal-footer">
-              <a data-dismiss="modal" data-toggle="modal" class="btn">cancel</a>
+              <a data-dismiss="modal" data-toggle="modal" class="btn">Cancel</a>
```
Comment (TildaDares) [inline index.html:68]: @Syed-Ansar These changes aren't supposed to be in this PR.

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by Syed-Ansar** [comment]
@TildaDares as said Cancel as cancel. Thank you.

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -11,6 +11,9 @@
     <link href="node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
     <link href="node_modules/font-awesome/css/font-awesome.min.css" rel="stylesheet">
 
+    <!-- Favicon -->
+    <link rel="icon" type="image/x-icon" href="/assets/favicon.ico">
+    <!-- CSS Link -->
```
Comment (TildaDares) [inline index.html:16]: This should be the only change in this PR.

File: index.html [hunk 3]
```diff
@@ -47,18 +50,18 @@ <h3>Quick start</h3>
...
                    <tr>
                      <td><b>NDVI for <span style="color:red;">RED</span> filters</b></td>
-                     <td>basic plant analysis for <a href="http://store.publiclab.org/products/infragram-point-shoot-plant-cam">Infragram Point &amp; Shoot cameras</a></td>
+                     <td>Basic plant analysis for <a target="_blank"  href="https://store.publiclab.org/collections/all-products/products/infragram-diy-filter-pack">Infragram DIY Filter Pack</a></td>
                      <td><a class="btn btn-primary" id="preset_ndvi_red">Basic</a>
                          <a class="btn btn-success" id="preset_ndvi_red_color">Colorized</a></td>
                    </tr>
                  </table>
-                 <p>Post a research note at <a href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
+                 <p>Post a research note at <a target="_blank"  href="http://publiclab.org">PublicLab.org</a> and email <a href="mailto:plots-dev@googlegroups.com">plots-dev@googlegroups.com</a> to suggest new presets.</p>
```
[the remark this answers is not in the document]
Comment (Syed-Ansar) [reply index.html:64]: Should I remove target-"_blank" ?

**Comment by Syed-Ansar** [comment]
New PR is made

**Pull request closed without merging**